app/src/main/java/co/nano/nanowallet/NanoUtil.java

```diff
@@ -250,8 +250,8 @@ public static String publicToAddress(String public_key) {
      * @return Public Key
      */
     public static String addressToPublic(String encoded_address) {
-        Sodium sodium = NaCl.sodium();
-        String data = encoded_address.substring(4, 56);
+        NaCl.sodium();
```

Comment:
Why this change?

Reply:
If you're asking about skipping the local `sodium` variable - it was not used, nor is required for Sodium to work. The `NaCl.sodium()` call can safely be done only once for the whole app, as it's responsible for loading native library and initializing it properly. Should either be done on application start, or in a `static` initializer of `NanoUtil`, no need to repeat it in every method.

Reply:
Oh, I thought we were calling methods on this variable.